docs/documentation/getting-started/configuration/index.markdown

```diff
@@ -158,6 +158,11 @@ The following variables are settable:
   * **default:** `0`
   * Number of seconds to wait after you reach the limit of concurrent connections to Git repository server and disconnect afterwards to initialize new connections. This prevents from being cut out of SSH server when you use `fail2ban` or similar software for limiting connections to server.
 
+* `:verify_commit`
```

Comment:
Hi @mohamedhafez so I'm more than happy to merge this if we change `:verify_commit` to `:git_verify_commit` in accordance with the existing `git` specific variables documented in this file (also at https://capistranorb.com/documentation/getting-started/configuration/)

That was the only issue I noticed, just consistency. I'm not sure other version control systems have a general purpose commit verification system, at least I searched a bit and didn't find anything, so I think we just focus on Git, for now?

Reply:
@leehambley done! made the changes and its passing all tests